charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 3.8.1
+version: 3.8.2
```

Comment:
I would suggest to bump the minor digit, since it changes a default behaviour

Reply:
Makes sense. Fixed!